ihmc-high-level-behaviors/src/main/java/us/ihmc/behaviors/activeMapping/ActivePlanarMappingRemoteTask.java

```diff
@@ -93,26 +93,23 @@ private void generalUpdate()
     */
    private void updateActiveMappingPlan()
    {
-      if (continuousPlanningParameters.getEnableContinuousHiking())
+      if (walkingStatusMessage.get() != null)
```

Comment:
Removed the check for the parameter that got deleted, and the rest of the changes in this file are formatting